### PR TITLE
Review1

### PR DESCRIPTION
**Pull request by NG90**

### CODE REVIEW

File: shm/cargo.cpp [hunk 1]
```diff
@@ -9,7 +9,12 @@ Cargo& Cargo::operator+=(size_t amount) {
     amount_ += amount;
     return *this;
 }
+
 Cargo& Cargo::operator-=(size_t amount) {
+    if (amount > amount_) {
+        std::cerr << "Not enough amount of cargo!\n";
```
Comment (kozlofski) [inline shm/cargo.cpp:15]: Maybe better: "Not enough amount of THIS cargo"

File: shm/island.cpp [hunk 2]
```diff
@@ -1,12 +1,11 @@
 #include "island.hpp"
 
-Island::Island(int position_X, int position_Y)
-    : position_(Coordinates(position_X, position_Y)) {
-}
```
Comment (kozlofski) [inline shm/island.cpp:5]: Delete new line between { }

File: shm/ship.hpp [hunk 3]
```diff
@@ -25,12 +25,11 @@ class Ship {
     std::vector<std::shared_ptr<Cargo>> getCargos() const { return cargo_; }
```
Comment (kozlofski) [inline shm/ship.hpp:25]: Newline after #pragma once
I cannot add comment at any of lines 1-25 :(

File: shm/cargo.cpp (hunk 1 above)
Comment (NG90) [reply shm/cargo.cpp]: don

File: shm/island.cpp (hunk 2 above)
Comment (NG90) [reply shm/island.cpp:5]: it's been already changed - done

File: shm/ship.hpp (hunk 3 above)
Comment (NG90) [reply shm/ship.hpp:26]: done

**Pull request merged**